packages/eslint-plugin/docs/rules/naming-convention.md

```diff
@@ -361,6 +361,23 @@ This allows you to emulate the old `interface-name-prefix` rule.
 }
 ```
 
+### Enforce that variable and function names are in camelCase
```

Comment:
haven't reviewed code yet - just a quick comment on the docs
Could you please document what an array of selectors means in the section above? (around line 157)

Reply:
thanks for letting me know this, not sure if this document https://github.com/typescript-eslint/typescript-eslint/pull/2335/files#diff-ae7b788bacbe18dbd048cb6522d14a33R158 is what you're expecting?

Reply:
perfect!